pkg/plugins/manager/signature/manifestverifier/verifier.go

```diff
@@ -30,17 +34,70 @@ type ManifestVerifier struct {
 	cfg  *config.Cfg
 	mlog log.Logger
 
-	lock       sync.Mutex
-	cli        http.Client
-	publicKeys map[string]ManifestKeys
+	lock sync.Mutex
+	cli  http.Client
+	kv   plugins.KeyStore
+}
+
+func New(cfg *config.Cfg, mlog log.Logger, kv plugins.KeyStore) *ManifestVerifier {
+	pmv := &ManifestVerifier{
+		cfg:  cfg,
+		mlog: mlog,
+		cli:  makeHttpClient(),
+		kv:   kv,
+	}
+	if pmv.featureEnabled() {
+		go func() {
+			pmv.updateKeysPeriodically()
+		}()
+	}
+	return pmv
+}
+
+func (pmv *ManifestVerifier) featureEnabled() bool {
+	return pmv.cfg != nil && pmv.cfg.Features != nil && pmv.cfg.Features.IsEnabled("pluginsAPIManifestKey")
```

Comment:
Nit: I would make this `"pluginsAPIManifestKey"` a const (inside `plugins` package).

There's also an instance of this literal in `loader_test.go` and `verified_test.go`